string request
---
string[] list_of_files
